### ISSUE TITLE
Change close button title / icon

### ISSUE THREAD

**Issue by DominikGanic** [fatihsolhan/v-onboarding#52]
Hi again, it would be great if we could somehow change the closing buttons title or content. As currently there is an SVG which is fine enough - but I'd like to modify the buttons class to set some bootstrap classes like "btn-primary btn-sm" which would give it some colors and style.

Also I'd like to place myself the content instead of SVG I would love to use the char "X" as inner value. It would be much better if we could also put some HTML inside like "<i class="fas fa-times"></i>" to integrate font awesome icons into the close button.

This would improve the entire behaviour as I'd have to hardcode it now and that's not a proper solution. We're planning to use this within a big spread and great platform for real estate rentals and leasing in germany - so we have to stick to some professional solutions instead of hardcoding anything.

Thanks mate! Hope to hear from you soon! :)

**Comment by fatihsolhan** [comment]
Hi @DominikGanic, isn't this easier with the custom UI?
Or is there a reason that you want to keep using the default UI?

**Comment by DominikGanic** [comment]
Yep, you're right. I'm currently implementing my custom UI. But I have problems with positionating the guide like previously popper did the job. Hm. Gonna need to figure out how I'll do this as it's now static.

**Comment by fatihsolhan** [comment]
Hey @DominikGanic, I would love to try helping with your case if you want to share your code examples

**Comment by DominikGanic** [comment]
Hi @fatihsolhan, watch out the screen below. There are two problems which make it complicated for me tho.

1. Popper is not setup here. The guide "wrapper" is static on top of my page.
2. The overlay is not visible, which always has been highlighting the element which I was showing off with my guide.

Sadly it's possible to determine the position of the currently attached element so I can't do anything here with a custom ui. Otherwise, using your UI, which was fine enough, I can't change the close button (icon as well).

That makes me sad. :(

![image](https://user-images.githubusercontent.com/13388387/218728103-3249d485-4a78-47c7-8edf-b8dcfc08d430.png)

**Comment by DominikGanic** [comment]
![image](https://user-images.githubusercontent.com/13388387/218728688-76ebd514-2148-41ae-886f-9b2008abbbdf.png)

Here you can see, the attached element on the left is being highlighted, while my wrapper is positionated absolutely on the page. That's perfect.

**Comment by DominikGanic** [comment]
I've managed to let it look like I've wanted. For anybody else, I've attached the CSS belowe which overrides some styles. The SVG with the "X" is also now perfectly visible. :)

![image](https://user-images.githubusercontent.com/13388387/218730953-fefb4008-0925-4614-8d85-aaae4dec101e.png)

`/* v-onboarding Customizations */
.v-onboarding-item__header-title {
  font-weight: 600 !important;
  color: #5598d2 !important;
}
.v-onboarding-item__description {
  color: #a1a5b7 !important;
  font-size: 12px !important;
  margin-top: unset !important;
}
.v-onboarding-item__header-close {
  font-size: 14px !important;
  color: white !important;
  background: #5598d2 !important;
  border-radius: 0.475rem !important;
  border: unset !important;
  margin: unset !important;
  padding: unset !important;
  box-shadow: unset !important;
  position: relative !important;
  height: 20px !important;
  width: 20px !important;
  transition: background 0.25s ease-in-out;

  &:hover {
    background: darken(#5598d2, 10%) !important;
  }

  svg {
    width: 12px;
    height: 12px;
  }
}`
